### ISSUE TITLE
Suppress single occurrences of MaxLineLength

### ISSUE THREAD

**Issue by benjamin-bader** [arturbosch/detekt#715]
There are certain constructs, e.g. very long URLs, that are not practical to shorten below any reasonable MaxLineLength.  As far as I can tell, we can suppress this rule with a file-level annotation only, but I'd prefer to use a finer-grained suppression.  Is this possible?

Ideally, I'd like to use something like:

```kt
@Suppress("MaxLineLength")
const val veryLongUrl = "http://blablabla"
```

**Comment by arturbosch** [comment]
Hmm, maybe something like:
```kotlin
lines.forEach { line ->
			offset += line.length
			if (!isValidLine(line)) {
				file.elementsInRange(TextRange.create(offset - line.length, offset))
						.find { it is KtAnnotatedExpression }
						?.let { report(CodeSmell(issue, Entity.from(it, offset), message = "")) }
						?: report(CodeSmell(issue, Entity.from(file, offset), message = ""))
			}

			offset += 1 /* '\n' */
		}
```
could work...

**Comment by schalkms** [comment]
@arturbosch can we commit this fix?

**Comment by arturbosch** [comment]
We can try this, if you like to open a PR go for it but also check the performance pls

**Comment by schalkms** [comment]
I copied the code and added the test case from above.
It didn’t work.

**Comment by gpeal** [comment]
@arturbosch This is fairly critical feature for us in a project with hundreds of thousands of lines of code. Could this be prioritized please?
